### PR TITLE
Fix metric for tinyllama_scale_estimation_per_channel

### PR DESCRIPTION
**Pull request by l-bat**
### Changes

* Accuracy bug with LLMs across different CPUs was fixed
* The problem was in execution of models from `get_compress_pipeline` and `get_compress_decompress_pipeline`
* TODO: add ticket link

### Related tickets

* 148819, 152627

### CODE REVIEW

**Review by alexsu52 — COMMENTED** [review]
Please provide results for different processors from CI.

**Comment by l-bat** [comment]
> Please provide results for different processors from CI.

11064, 11066

**Review by alexsu52 — APPROVED** [review]
LGTM

File: tests/post_training/data/wc_reference_data_2024.5.yaml [hunk 1]
```diff
@@ -3,12 +3,8 @@ tinyllama_data_aware_gptq_backend_OV:
...
-tinyllama_scale_estimation_per_channel_backend_OV:
-  metric_value: 0.80798
-  num_int4: 188
-  num_int8: 124
-  metrics_xfail_reason: "Issue-148819"
 tinyllama_NF4_scale_estimation_stateful_per_channel_backend_OV:
   metric_value: 0.87132
   num_int4: 11
   num_int8: 290
+  metrics_xfail_reason: "Issue-148819"
```
Comment (alexsu52) [inline tests/post_training/data/wc_reference_data_2024.5.yaml:10]: Why did you add xfail for tinyllama_NF4_scale_estimation_stateful_per_channel_backend_OV?

Comment (l-bat) [reply tests/post_training/data/wc_reference_data_2024.5.yaml]: Good catch! I've removed it.

**Review by alexsu52 — APPROVED** [review]
